qutip/wigner.py

```diff
@@ -574,31 +574,57 @@ def _qfunc_check_state(state: Qobj):
     return state
 
 
-def _qfunc_check_coordinates(xs, ys):
-    if np.isscalar(xs) or xs is None:
-        raise TypeError("xs must be array-like, but is " + repr(xs))
-    if np.isscalar(ys) or ys is None:
-        raise TypeError("ys must be array-like, but is " + repr(ys))
-    xs = np.asarray(xs, dtype=np.float64)
-    ys = np.asarray(ys, dtype=np.float64)
-    if xs.ndim != 1 or ys.ndim != 1:
+def _qfunc_check_coordinates(xvec, yvec):
+    if np.isscalar(xvec) or xvec is None:
+        raise TypeError("xvec must be array-like, but is " + repr(xvec))
+    if np.isscalar(yvec) or yvec is None:
+        raise TypeError("yvec must be array-like, but is " + repr(yvec))
+    xvec = np.asarray(xvec, dtype=np.float64)
+    yvec = np.asarray(yvec, dtype=np.float64)
+    if xvec.ndim != 1 or yvec.ndim != 1:
         raise ValueError(
-            f"xs and ys must be 1D, but have shapes {xs.shape} and {ys.shape}."
+            f"xvec and yvec must be 1D, but have shapes {xvec.shape} and {yvec.shape}."
         )
-    return xs, ys
+    return xvec, yvec
 
 
-class _qfunc_alpha_matrix:
+class _CoherentGrid:
     """
-    Internal class used to centrally manage the creation of tensors related to
-    coherent states used in the calculation of the Husimi-Q function.
+    Internal class to compute coherent state operators corresponding to a grid of complex
+    values in the phase space. A call to an object of this class gives the same results as
+    `qutip.coherent` for the predefined grid of xvec, yvec.
+
+    Example
+    --------
+    >> N = 64 # Hilbert size
+    >> xvec, yvec, g = [1, 0, -1], [1, 0, -1], 1 # Grid and coefficient g
+
+    >> X, Y = np.meshgrid(xvec, yvec)
+    >> betas = 0.5*g*(X - 1j*Y).ravel() # Complex numbers denoting phase space points
+
+    QuTiP's implementation of computing coherent states
+    >> coherent_states_qutip = np.array([coherent(N, beta).full() 
+                                    for beta in betas]).reshape(3, 3, -1) # reshaped to a tensor
```

Comment:
Looks like you're using `black` with too high a line limit for QuTiP.  QuTiP's got a hard limit at 79 characters, rather than black's lax limit up to 88.